.env

```diff
@@ -0,0 +1,3 @@
+REDIS_URL="redis://localhost:6379/1"
+ALLOWED_CORS_ORIGINS='*'
+RAILS_SERVE_STATIC_FILES='true'
```

Comment:
Do not add the .env file to the repository.

Reply:
Has been removed and added to .gitignore